packages/button/src/Button/components/GenericButton.tsx

```diff
@@ -17,7 +17,7 @@ import styles from "./GenericButton.scss"
 export type CustomButtonProps = {
   id?: string
   className: string
-  ref: Ref<any>
+  forwardRef: Ref<any>
```

Comment:
Hmm... I would expect the fix to actually be using React's `forwardRef` then usage using `ref` as opposed to having a custom prop named `forwardRef`. Is this really the right direction? 😬

Reply:
It's a similar solve to the problem discussed in the react docs here for Higher-order-components: https://reactjs.org/docs/forwarding-refs.html#forwarding-refs-in-higher-order-components

The Button is a forward ref-able component, but to forward the ref _again_ to a React component, it needs to pass forward in this manner.

Reply:
I'd suggest we change the prop to `forwardedRef` tho :P